gbpservice/nfp/service_plugins/vpn/drivers/nfp_vpnaas_driver.py

```diff
@@ -256,3 +253,6 @@ def create_vpnservice(self, context, vpnservice):
             rsrc_id=vpnservice['id'],
             resource=vpnservice,
             reason='create', service_vendor=service_vendor)
+
+    def delete_vpnservice(self, context, vpnservice):
```

Comment:
actually we get call delete-vpn-service from heat engine, because of which we were getting that extra call with empty kwargs which breaking in config_orch's vpn agent so here we are suppressing that.